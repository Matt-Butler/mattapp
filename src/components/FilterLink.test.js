import React from 'react';
import { shallow } from 'enzyme';
import FilterLink from './FilterLink';

it('renders without crashing', () => {
  const wrapper = shallow(
    <FilterLink/>
  );
  expect(wrapper).toMatchSnapshot();
});