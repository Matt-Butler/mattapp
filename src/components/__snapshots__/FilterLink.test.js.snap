// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<NavLink
  activeStyle={
    Object {
      "color": "black",
      "textDecoration": "none",
    }
  }
  exact={true}
  to="/undefined"
/>
`;
